ddtrace/tracer/option.go

```diff
@@ -102,6 +102,34 @@ func defaults(c *config) {
 			log.Warn("unable to look up hostname: %v", err)
 		}
 	}
+
+	if os.Getenv("DD_TRACE_GLOBAL_TAGS") != "" {
+
+		if c.globalTags == nil {
+			c.globalTags = make(map[string]interface{})
+		}
+
+		osTags := os.Getenv("DD_TRACE_GLOBAL_TAGS")
+		if osTags != "" {
+			for _, tag := range strings.Split(osTags, ",") {
+				if tag != "" {
+					tag = strings.TrimSpace(tag)
+					var keyVal = strings.Split(tag, ":")
```

Comment:
We don't really use this style in Go. Can you please change to the below and remove the previous line:
```suggestion
					kv := strings.SplitN(strings.TrimSpace(tag), ":", 2)
```
Use `SplitN` to allow colons to live in the tag value.